README.md

```diff
@@ -43,9 +43,10 @@ The following must be installed and available in your `$PATH`:
 One simple way to run this without installing PhantomJS on your local machine is to use the included DockerFile.
 
 ```bash
-docker build -t ge-cancellation-checker .
+git clone https://github.com/davidofwatkins/ge-cancellation-checker.git
```

Comment:
Hmm, I think it's understood that the repo is already cloned at this point, no? Maybe preface these instructions with a note about cloning if it hasn't been done already, like so:

```bash
# If not already cloned:
git clone https://github.com/davidofwatkins/ge-cancellation-checker.git
cd ge-cancellation-checker

# run docker:
cp config.json.example config.json
# ...

```